alacritty/src/display/mod.rs

```diff
@@ -406,6 +411,18 @@ impl Display {
         });
     }
 
+    fn update_resize_increments(&self, config: &Config) {
+        self.window.update_resize_increments(if config.ui_config.window.resize_increments {
+            let cell_size = PhysicalSize::<f32>::from((
+                self.size_info.cell_width(),
+                self.size_info.cell_height(),
+            ));
+            Some(cell_size.to_logical(self.window.dpr))
+        } else {
+            None
+        });
```

Comment:
Please never do this. The goal of patches should be to produce readable, not compressed code. Please extract these conditionals into separate statements.

Reply:
I actually never realized that `bool::then` was stabilized, I'll refactor this

Reply:
`bool::then` is not an option due to our MSRV.

Reply:
Now, a year later, it is :)